src/controller/authentication-controller.ts

```diff
@@ -15,18 +15,18 @@
  *  You should have received a copy of the GNU Affero General Public License
  *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
-import { Request, Response } from 'express';
-import log4js, { Logger } from 'log4js';
-import BaseController, { BaseControllerOptions } from './base-controller';
```

Comment:
Why are all those spaces deleted here? If I remember correctly, the ESLint wants these spaces here? Weirdly enough the CI does not complain about it, but it is very inconsistent with the rest of the codebase.